packages/wasm-dpp/test/unit/stateTransition/validation/validateStateTransitionIdentitySignatureFactory.spec.js

```diff
@@ -227,10 +228,13 @@ describe('validateStateTransitionIdentitySignatureFactory', () => {
 
       expect(error).to.be.instanceOf(InvalidSignaturePublicKeySecurityLevelError);
       expect(error.getPublicKeySecurityLevel()).to.equal(IdentityPublicKey.SECURITY_LEVELS.MASTER);
-      expect(error.getKeySecurityLevelRequirement()).to.equal(2);
+      expect(error.getKeySecurityLevelRequirement()).to.deep.equal([2]);
```

Comment:
Why is this an array now?

Reply:
This is an array because it just makes more sense, you set the security level requirements that you want, instead of saying it has to be higher than something. Before we had something like higher than "High", but not Master, now instead you would just have "High" + "Critical". This reduced the chance of bugs in the future as you are directly saying the allowed security levels.

Reply:
I also allows for things like Medium only which wasn't possible before.

Reply:
Okay, makes sense, more granularity. Thanks for the response 👍